src/frontend/src/btc/components/send/BtcSendHasPendingTransactions.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+	import { fade } from 'svelte/transition';
+	import WarningBanner from '$lib/components/ui/WarningBanner.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let pendingTransactionsStatus: boolean | 'loading' | 'error';
+</script>
+
+<div in:fade>
```

Comment:
the `div` should be within the ifs. so for each `if` a `div in:fade`.

Reply:
Oops, done!

Reply:
done